features/generators.feature

```diff
@@ -10,16 +10,22 @@ Feature:
     And I run `bundle install` with a clean environment
 
   Scenario: The factory_bot_rails generators create a factory file for each model if there is not a factories.rb file
-    When I run `bundle exec rails generate model User name:string age:integer` with a clean environment
+    When I run `bundle exec rails generate model User name:string age:integer company:belongs_to email home_url password password_confirmation position` with a clean environment
     And I run `bundle exec rails generate model Namespaced::User name:string` with a clean environment
     Then the output should contain "test/factories/users.rb"
     And the output should contain "test/factories/namespaced/users.rb"
     And the file "test/factories/users.rb" should contain exactly:
       """
       FactoryBot.define do
         factory :user do
-          name { "MyString" }
-          age { 1 }
+          sequence(:name) {|n| "Name#{format '%03d', n}" }
+          sequence(:age) {|n| "1#{format '%03d', n}" }
+          association :company, factory: :company
```

Comment:
Auto-generating associations sounds like a good improvement.

What are your thoughts on splitting this out into its own PR?

Reply:
this code is directly connected with the changes in lib/generators/factory_bot/model/model_generator.rb and covers it with tests

Reply:
Sure. I suggest pulling this line, the applicable code in `model_generator.rb`, and the corresponding test, and breaking it out into a separate PR.